crates/bevy_reflect/src/lib.rs

```diff
@@ -796,6 +796,169 @@ mod tests {
         assert_eq!(values, vec![1]);
     }
 
+    #[test]
+    fn should_reflect_clone() {
+        // Struct
+        #[derive(Reflect, Clone, Debug, PartialEq)]
+        #[reflect(Clone)]
+        struct Foo(usize);
+
+        let value = Foo(123);
+        let clone = value.reflect_clone().expect("should reflect_clone struct");
+        assert_eq!(value, clone.take::<Foo>().unwrap());
+
+        // Generic Struct
+        #[derive(Reflect, Debug, PartialEq)]
+        struct Bar<T, U>(T, #[reflect(ignore, clone)] PhantomData<U>);
+        #[derive(TypePath, Debug, PartialEq)]
+        struct Baz;
+
+        let value = Bar::<usize, Baz>(123, PhantomData);
+        let clone = value
```

Comment:
so here the clone uses the `<usize as dyn Reflect>::clone_value` function?

Reply:
Good point, will do!